Explorer/Assets/DCL/Multiplayer/SDK/Systems/WriteAvatarEquippedDataSystem.cs

```diff
@@ -0,0 +1,102 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using CommunicationData.URLHelpers;
+using CRDT;
+using CrdtEcsBridge.ECSToCRDTWriter;
+using DCL.Diagnostics;
+using DCL.ECSComponents;
+using DCL.Multiplayer.SDK.Components;
+using DCL.Optimization.Pools;
+using ECS.Abstract;
+using ECS.LifeCycle.Components;
+
+namespace DCL.Multiplayer.SDK.Systems
+{
+    [UpdateInGroup(typeof(PresentationSystemGroup))]
```

Comment:
All scene related systems should be in synchronized groups

Reply:
Updated them to `SyncedPostRenderingSystemGroup`